src/Utils/InstrumentSchema.php

```diff
@@ -9,6 +9,17 @@
 use WPGraphQL\AppContext;
 use WPGraphQL\Type\WPObjectType;
 
+/**
+ * Create unique custom "nil" value which is different from the build-in PHP
+ * null, false etc. When this custom "nil" is returned from a filter we can
+ * know that the filter did not try to return anything custom because it does
+ * not equal with anything but itself.
+ *
+ * Exposing this as a global so it can be accessed even if other overloaded
+ * filters return something else.
+ */
+$graphql_nil = new \stdClass();
```

Comment:
Instances like this cannot be assigned to constants or static class properties so the only way to expose this is a global variable.